locale/en-us.yaml

```diff
@@ -1248,6 +1248,14 @@ package_version_updated:
   other: "Package updated: {{.V0}}@{{.V1}}"
 package_ingredient_err:
   other: Failed to resolve an ingredient named {{.V0}}.
+package_headless_added:
+  other: "Package added: {{.V0}}"
+package_headless_updated:
+  other: "Package updated: {{.V0}}"
+package_headless_removed:
+  other: "Package removed: {{.V0}}"
```

Comment:
Are these still correct with us relying on the swagger?

Reply:
Confirmed: https://github.com/ActiveState/cli/blob/master/pkg/platform/api/mono/mono_models/commit_change_editable.go#L72-L78